### DISCUSSION TITLE
Android Webview DNS

### DISCUSSION THREAD

**Discussion by DionGanteng** [community/community#30713] [Other Feature Feedback, Questions, & Ideas]
How to set dns on webview? 
have searched various sources but it doesn't work

**Comment by jfullstackdev** [comment]
not sure if this will help 

https://supportportal.juniper.net/s/article/Archive-WLC-Enabling-and-configuring-DNS-by-using-WLC-WebView?language=en_US

**Comment by mmqeth** [comment]
https://www.youtube.com/watch?v=MpAIw9EKafM

This is the general solution
